tests/pom/admin/Admin.ts

```
@@ -0,0 +1,59 @@
+
+  static goToInstalledElementsList(
+    page: Page,
+    options: { validate: boolean } = { validate: false },
+  ) {
+    return AdminSection.visit(
+      page,
+      AdminSection.installedElementsRoute,
+      options.validate,
+    );
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)